tests/framework/integration.go

```diff
@@ -315,6 +394,18 @@ func getOps(ss []string) ([]clientv3.Op, error) {
 	return ops, nil
 }
 
+func (c integrationClient) MemberAdd(ctx context.Context, _ string, peerAddrs []string) (*clientv3.MemberAddResponse, error) {
```

Comment:
Any reason you reorder the functions here? It only obscures git history.

Reply:
Oh, I wanted it to be in the same order of https://github.com/etcd-io/etcd/blob/cd9764a99fab81788d6234d3524a08c9b893c3e4/tests/framework/interface.go#L72-L79

Reply:
Please separate logic changes from code cleanups, either separate commits or PRs.

Reply:
Make sense. Will do